modules/cnf-about-nic-support-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_nic_support_irq_affinity_setting_{context}"]
+= About NIC support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask.
+
+The following list by no means exhaustive are examples of drivers and hardware that Red Hat are aware lack support for IRQ re-balancing:
+
+* Some RAID controller drivers, such as `megaraid_sas`
+* Many non-volatile memory express (NVMe) drivers
+* Some LAN on motherboard (LOM) network controllers
+
+[NOTE]
+====
+LOMs are actually NICs controlled by NIC drivers.
+The reason they do not support IRQ affinity setting, is probably associated with factors such as the type of processor, the IRQ controller, or the circuitry connections in the motherboard.
+====
+
+These IRQ controllers effectively run on CPU 0. To find the effective affinity, log in to the host and run the following:
```

Comment:
I would move the first sentence, as I already pointed above.
Maybe I would mention here something like "If the effective affinity of any IRQ is set to an isolated CPU, it MIGHT be a sign of some hardware or driver not supporting IRQ affinity setting"